workflows/charts/tgi/values.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -- Name of the serving service
+nameOverride: ""
+# -- Full qualified Domain Name
+fullnameOverride: ""
+deploy:
+  # -- ConfigMap of Environment Variables
+  configMapName: intel-proxy-config
```

Comment:
I assume this ConfigMap is only available internally...

Reply:
It's the name of an object that can contain any amount of environment variables. Users can configure the chart to use their own or mark it as nil